roxie/ccd/ccdfile.cpp

```diff
@@ -37,6 +37,10 @@
 #ifdef __linux__
 #include <sys/mman.h>
 #endif
+#if defined (__linux__) || defined (__APPLE__)
+#include <sys/syscall.h>
+#include <ioprio.h>
```

Comment:
Debatable... I'm not sure we are consistent about the distinction anyway, but ioprio.h is really a system include that we just happen to have to ship because it's part of the linux kernel sources...